contributing/building-an-amp-extension.md

```diff
@@ -679,20 +680,20 @@ Class AmpMyElement extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
-    if (!isExperimentOn(this.getWin(), EXPERIMENT)) {
-      user.warn('Experiment %s is not turned on.', EXPERIMENT);
-      return;
-    }
+    if(!isExperimentOn(this.win, 'amp-my-element')) {
```

Comment:
Nit: here and below, please add a space after `if`. Also, is there any reason why you no longer want to use the constant `EXPERIMENT`?

Reply:
Yes -- passing the constant fails lint with error `isExperimentOn must be passed an explicit string`

Reply:
Oh... Does adding `@const {string}` to the `EXPERIMENT` constant fix it?

Reply:
No, I tried that also. I wonder why this check is enforced this way...

Reply:
Thanks for checking :))